Automattic/wp-calypso
# Paid Stats: Update gating overlay copy and gate author stats

Fixes https://github.com/Automattic/dotcom-forge/issues/4955

## Proposed Changes

| Before  | After |
| :-------------: | :-------------: |
| <img src="https://github.com/Automattic/wp-calypso/assets/6586048/0f0413b6-474c-49d6-b414-b5b74739e099">  | <img src="https://github.com/Automattic/wp-calypso/assets/6586048/743f98a0-8118-41f7-a907-ffebdcdc9f80">|

## Testing Instructions



* On a free site, go to `/stats` with the flag `?flags=stats/paid-wpcom-v2`
* Check the copy to see it can be enhanced

## Pre-merge Checklist
